audnauseum/gui/ui.py

```diff
@@ -204,3 +209,67 @@ def show_popup(ui):
     msg.setText("A loop must be loaded.")
 
     msg.exec_()
+
+
+def transport_status(ui, looper: Looper, status):
+
+    if not looper.state == LooperStates.IDLE:
+
+        if status == 'record':
+
+            ui.status_indicator.setStyleSheet("""
+                                                QPushButton 
+                                                {
+                                                    color: #333;
+                                                    border: 2px solid #555;
+                                                    border-radius: 20px;
+                                                    border-style: outset;
+                                                    background: qradialgradient(
+                                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
+                                                        radius: 1.35, stop: 0 #fff, stop: 1 #b12
+                                                        );
+                                                    padding: 5px;
+                                                }
+                                                """
+                                              )
```

Comment:
Possibly in the future splitting these stylesheets out to a separate file or format, but totally fine with this approach for now given our timeline.